CMakeLists.txt

```diff
@@ -152,6 +161,23 @@ pbuilder_install_files(${CMAKE_INSTALL_PREFIX}/bin/rootwc-files
         ${PROJECT_SOURCE_DIR}/rootwc/rootwc.C
         )
 
+pbuilder_install_files(${CMAKE_INSTALL_PREFIX}/mPMT-configfiles
+        ${PROJECT_SOURCE_DIR}/mPMT-configfiles/mPMTconfig_19_nuPrism.txt
+        ${PROJECT_SOURCE_DIR}/mPMT-configfiles/mPMTconfig_19_nuPrism_3ring.txt
+        ${PROJECT_SOURCE_DIR}/mPMT-configfiles/mPMTconfig_30_13_3.txt
+        ${PROJECT_SOURCE_DIR}/mPMT-configfiles/mPMTconfig_33_13_1.txt
+        ${PROJECT_SOURCE_DIR}/mPMT-configfiles/mPMTconfig_33_50.txt
+        ${PROJECT_SOURCE_DIR}/mPMT-configfiles/mPMTconfig_33_50_1.txt
+        ${PROJECT_SOURCE_DIR}/mPMT-configfiles/mPMTconfig_34_22.5_1.txt
+        ${PROJECT_SOURCE_DIR}/mPMT-configfiles/mPMTconfig_34_22.5_2.txt
+        ${PROJECT_SOURCE_DIR}/mPMT-configfiles/mPMTconfig_34_22_1.txt
+        ${PROJECT_SOURCE_DIR}/mPMT-configfiles/mPMTconfig_34_22_2.txt
+        ${PROJECT_SOURCE_DIR}/mPMT-configfiles/mPMTconfig_37_HK.txt
+        ${PROJECT_SOURCE_DIR}/mPMT-configfiles/mPMTconfig_Position_WCTE.txt
+        ${PROJECT_SOURCE_DIR}/mPMT-configfiles/mPMTconfig_Position_WCTE_RotateBarrelHalfTower.txt
+        )
```

Comment:
General question for the CMakeLists.txt @guiguem Is there a reason why installing files is done on a per file basis, rather than just installing everything in a folder? The latter is possible - see for example [this answer](https://stackoverflow.com/a/67883070)

Reply:
Yeah this is a tricky case for config files. If people forget to add/remove them for `CMakeLists.txt` for compiled code, it's obvious. Not so for config files. But as you say, it's easier to add/remove them as required if done explicitly like this.

Happy to keep it as it is for WCSim [mostly because I haven't formed a preference yet], and maybe think again for GHOST & other Tool packages